lib/dry/events/publisher.rb

```diff
@@ -236,6 +245,20 @@ def process(event_id, payload = EMPTY_HASH, &block)
         def __bus__
           @__bus__ ||= self.class.new_bus
         end
+
+        # Utility method that check that the event has been registered
+        #
+        # @return [Boolean]
+        #
+        # @api private
+        def registered_event?(object_or_event_id)
+          case object_or_event_id
+          when String, Symbol
+            __bus__.register_event?(object_or_event_id)
+          else
+            true
+          end
+        end
```

Comment:
Not sure if it's OK that this returns `true` for other types?

Reply:
Since the `subscribe` method accepts either an object listener or an event name we need to do a check here, I agree with the concern of returning always `true` for no event name, maybe we could move the check up to the `subscribe` method and only call this method if is an event name.